app/javascript/app/components/cards/cards.js

```diff
@@ -0,0 +1,11 @@
+import { connect } from 'react-redux';
+import { getCardsData } from './cards-selectors';
+
+import CardsComponent from './cards-component';
+
+const mapStateToProps = ({ countriesOverviewData }) => {
```

Comment:
What about using this: https://github.com/reduxjs/reselect#createstructuredselectorinputselectors-selectorcreator--createselector
as for example here: https://github.com/ClimateWatch-Vizzuality/south-africa-platform/blob/560b7f3ad141d68189a45962bcd59717c832932f/app/javascript/app/components/total-ghg-emissions/total-ghg-emissions-selectors.js#L153